rust_team_data/src/v1.rs

```diff
@@ -125,11 +125,20 @@ pub struct ZulipGroups {
 
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 pub struct Permission {
+    pub people: Vec<PermissionPerson>,
     pub github_users: Vec<String>,
     pub github_ids: Vec<usize>,
     pub discord_ids: Vec<usize>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
+pub struct PermissionPerson {
+    pub github_id: usize,
+    pub github: String,
+    pub name: String,
+    pub discord_id: Option<usize>,
```

Comment:
I wonder if we could leave this out unless we're sure we need it. At some point I'd like to review whether we actually need discord support and that would be made easier by not adding more references to discord.

Reply:
https://github.com/rust-lang/team/pull/1196#discussion_r1448944393 🤷‍♂️ 

I'm fine with removing it if we're sure that nothing is actually using it

Reply:
At the very least, no one is *definitely* using it through the `people` field :laughing:

Reply:
I've removed the field now